core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -130,10 +155,24 @@ void append() throws CommitFailedException {
                         row.cancel();
                         row = null;
                         final int colType = defaultColumnTypes.MAPPED_COLUMN_TYPES[entityType];
-                        writer.addColumn(columnName, colType);
+                        // we have to commit before adding a new column as WalWriter doesn't do that automatically
+                        writer.commit();
+                        try {
+                            addColumnBuilder.clear();
+                            addColumnBuilder.ofAddColumn(0, tableUpdateDetails.getTableNameUtf16(), 0);
```

Comment:
why is there two calls to `ofAddColumn()` ?

Reply:
The methods are different in spite of the same name. The first one defines the table name while the second one sets column info. We have the same for `AlterOperationBuilder#ofRenameColumn()`. Do you think we should rename them?

Reply:
could you check usages of 

```
ofAddColumn(int tableNamePosition, String tableName, int tableId)
```

? 

There are 7. 6 out of 7 use pair of `ofColumn` calls like this one. We should have one call.

Reply:
Let me check and try to merge the methods.

Reply:
It appears that the methods should be kept separate, as add column and partition statements support multiple columns/partitions. Renamed the methods to `addColumnToList`/`addPartitionToList` in 782fbf26cd6b864c884814b88567154ca73fdf1e to make them more obvious.